backend/protocol_rpc/rpc_methods.py

```
@@ -163,9 +165,11 @@ async def update_validator(
     plugin: str | None = None,
     plugin_config: dict | None = None,
     session: Session = Depends(get_db_session),
+    validators_manager=Depends(get_validators_manager),
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)